andela/AndelaEatsUI
# #160251052: Fix Pre-Commit Hook setup

#### What does this PR do?
Fixes pre-commit hook by replacing pre-commit module with husky

#### Description of Task to be completed?

#### How should this be manually tested?
- Run `npm i` to install husky package
- Check out a new branch
- Try to make a commit with a bad commit message: it should flag it
- Also, linter and test should be run before the pushing is allowed.

#### What are the relevant pivotal tracker stories?
- [#160251052](https://www.pivotaltracker.com/story/show/160251052)

#### Background Context

#### Screenshots (If Applicable)
